e2e/__tests__/__snapshots__/toMatchInlineSnapshot.test.ts.snap

```diff
@@ -120,6 +120,22 @@ test('handles property matchers', () => {
 
 `;
 
+exports[`indentation is correct in the presences of existing snapshots: existing snapshot 1`] = `
+test('existing snapshot', () => {
+  expect({hello: 'world'}).toMatchInlineSnapshot(\`
```

Comment:
I'm not sure why this gets indented, but it doesn't seem to be related to the code change so I think it's harmless?

Reply:
`dedent` fixed it 👍